### PR TITLE
Cleanup meta

### PR DESCRIPTION
**Pull request by jschueller**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge/core is there a problem with the linter bot ? my patch seems to build the recipe fine

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,35 +12,29 @@ source:
 build:
   number: 2
```
Comment (scopatz) [inline recipe/meta.yaml]: should bump build number

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,35 +12,29 @@ source:
...
   skip: true  # [win]
-  features:
```
Comment (scopatz) [inline recipe/meta.yaml:22]: we should probably add run exports too

File: recipe/meta.yaml (hunk 1 above)
Comment (jschueller) [reply recipe/meta.yaml]: no, currently it does not change the content of the package

File: recipe/meta.yaml (hunk 2 above)
Comment (jschueller) [reply recipe/meta.yaml:22]: I've never used run exports, what should it look like ?
```
build:
  run_exports:
    - libclp
    - libcbc
    ...
```

Comment (scopatz) [reply recipe/meta.yaml:22]: from https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements, it should look something like:

```yaml
build:
  run_exports:
    - {{ pin_subpackage('libpng', max_pin='x.x') }}
```

Comment (jschueller) [reply recipe/meta.yaml:22]: ok

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**